bindings/python/.evergreen/test.sh

```diff
@@ -42,14 +50,23 @@ else
              "/opt/python/3.6/bin/python3"
              "/opt/python/pypy/bin/pypy"
              "/opt/python/pypy3.6/bin/pypy3")
+
+    ${PYTHONS[${#PYTHONS[@]}-3]} drivers-evergreen-tools/.evergreen/mongodl.py --component \
+    csfle --version 6.0.0-rc4 --out ~/csfle/ --target rhel70
 fi
 
+
 for PYTHON_BINARY in "${PYTHONS[@]}"; do
     echo "Running test with python: $PYTHON_BINARY"
     $PYTHON_BINARY -c 'import sys; print(sys.version)'
     createvirtualenv $PYTHON_BINARY .venv
     python -m pip install --prefer-binary -r test-requirements.txt
+    DYLD_FALLBACK_LIBRARY_PATH=~/csfle/lib/:$DYLD_FALLBACK_LIBRARY_PATH \
+    LD_LIBRARY_PATH=~/csfle/lib:$LD_LIBRARY_PATH \
+    PATH=~/csfle/bin:$PATH \
     python setup.py test
+    python setup.py test -s test.test_mongocrypt.TestMongoCrypt
     deactivate
     rm -rf .venv
 done
+rm -rf ~/csfle/
```

Comment:
Would it makes sense to put the library into the main evergreen checkout directory, i.e. the one above the `libmongocrypt` checkout? That one gets cleaned up automatically on failure, the home directory does not (at least not on all hosts).